package.json

```diff
@@ -3,18 +3,43 @@
   "version": "1.0.0",
   "description": "dxw Playbook",
   "scripts": {
-    "start": "docsify serve .",
-    "lint": "remark .",
+    "markdown:lint": "remark .",
     "vale": "~/go/bin/vale */*.md *.md",
-    "vale-suggestions": "~/go/bin/vale --minAlertLevel suggestion */*.md *.md "
+    "vale-suggestions": "~/go/bin/vale --minAlertLevel suggestion */*.md *.md ",
+    "build:dev": "webpack --mode=development",
+    "build": "webpack -p --mode=production",
+    "watch": "webpack --mode=development --watch",
+    "js:lint": "standard '_javascripts/src/**/.js' '_spec/*.js'",
+    "js:lint:fix": "standard '_javascripts/src/**/.js' '_spec/*.js' --fix",
+    "prettier": "prettier --check '_javascripts/src/**/*.js' '_spec/*.js'",
+    "prettier:fix": "prettier --write '_javascripts/src/**/*.js' '_spec/*.js'",
+    "test": "jest"
   },
   "dependencies": {
```

Comment:
Let's install `husky` to automatically run lint and formatting fixes pre-commit.

Reply:
Will the hooks run and work for users who edit in the browser though?

Reply:
No, but it's better than nothing.

Reply:
Done! 👍